### PR TITLE
CNV 2.6 release notes: 2.5 known issues

### PR DESCRIPTION
**Pull request by ousleyp**
* Part of [CNV-7094](https://issues.redhat.com/browse/CNV-7094)
* This PR also changes version numbers for [CNV-10319](https://issues.redhat.com/browse/CNV-10319) but these will need to be rechecked

### CODE REVIEW

File: virt/virt-2-6-release-notes.adoc [hunk 1]
```diff
@@ -85,19 +85,8 @@ Other operating system templates shipped with {VirtProductName} are not supporte
...
-
-* If you enable a MAC address pool for a namespace by applying the KubeMacPool label and using the `io` attribute for virtual machines in that namespace, the `io` attribute configuration is not retained for the VMs. As a workaround, do not use the `io` attribute for VMs. Alternatively, you can disable KubeMacPool for the namespace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1869527[*BZ#1869527*])
-
-* If you upgrade to {VirtProductName} {VirtVersion}, both older and newer versions of common templates are available for each combination of operating system, workload, and flavor. When you create a virtual machine by using a common template, you must use the newer version of the template. Disregard the older version to avoid issues. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1859235[*BZ#1859235*])
+//This bug is still on New status. Is this still relevant?
 
 * Running virtual machines that cannot be live migrated might block an {product-title} cluster upgrade. This includes virtual machines that use hostpath-provisioner storage or SR-IOV network interfaces. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1858777[*BZ#1858777*])
```
Comment (ousleyp) [inline virt/virt-2-6-release-notes.adoc:91]: @stu-gott Do you know if this release note should be retained in 2.6?

File: virt/virt-2-6-release-notes.adoc [hunk 2]
```diff
@@ -106,9 +95,9 @@ As a workaround, you can reconfigure the virtual machines so that they can be po
...
 . Set the `runStrategy` field to `Always`.
 
-* For unknown reasons, memory consumption for the `containerDisk` volume type might gradually increase until it exceeds the memory limit. To resolve this issue, restart the VM. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1855067[*BZ#1855067*])
+//Is this still relevant with the UI changes (the bug itself has not been fixed)? If it is still relevant, how should we change the patch command to reflect the 2.6 version?
 
-* Sometimes, when attempting to edit the subscription channel of the *{VirtProductName} Operator* in the web console, clicking the *Channel* button of the *Subscription Overview* results in a JavaScript error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1796410[*BZ#1796410*])
+* Sometimes, when attempting to edit the subscription channel of the {VirtProductName} Operator in the web console, clicking the *Channel* button of the *Subscription Overview* results in a JavaScript error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1796410[*BZ#1796410*])
```
Comment (ousleyp) [inline virt/virt-2-6-release-notes.adoc:100]: @tiraboschi Do you know if this known issue should be retained for 2.6 (see comment)?

File: virt/virt-2-6-release-notes.adoc [hunk 3]
```diff
@@ -147,11 +138,13 @@ data:
 ----
 <1> Replace `<cpu-model>` with the actual CPU model value. You can determine this value by running `oc describe node <node>` for all nodes and looking at the `cpu-model-<name>` labels. Select the CPU model that is present on all of your nodes.
 
+// No bug attached to this -- is it still relevant?
+
 * {VirtProductName} cannot reliably identify node drains that are triggered by running either `oc adm drain` or `kubectl drain`. Do not run these commands on the nodes of any clusters where {VirtProductName} is deployed. The nodes might not drain if there are virtual machines running on top of them.
 
 ** The current solution is to xref:../virt/node_maintenance/virt-setting-node-maintenance.adoc#virt-setting-node-maintenance[put nodes into maintenance].
```
Comment (ousleyp) [inline virt/virt-2-6-release-notes.adoc:145]: @stu-gott Do you know if this should be retained for 2.6?

File: virt/virt-2-6-release-notes.adoc [hunk 4]
```diff
@@ -147,11 +138,13 @@ data:
...
 
-* If the {VirtProductName} storage PV is not suitable for importing a RHV VM, the progress bar remains at 10% and the import does not complete. The VM Import Controller Pod log displays the following error message: `Failed to bind volumes: provisioning failed for PVC`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1857784[*BZ#1857784*])
+// This was closed WONTFIX, is it still relevant?
 
 * If you enter the wrong credentials for the RHV Manager while importing a RHV VM, the Manager might lock the admin user account because the `vm-import-operator` tries repeatedly to connect to the RHV API. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1887140[*BZ#1887140*])
```
Comment (ousleyp) [inline virt/virt-2-6-release-notes.adoc:149]: @pkliczewski Do you know if this release note should be retained in 2.6 (see comment)?

**Review by ousleyp — COMMENTED** [review]
Some questions for SMEs. If I have tagged the wrong person, please let me know or tag someone else. Thank you!

File: virt/virt-2-6-release-notes.adoc (hunk 4 above)
Comment (pkliczewski) [reply virt/virt-2-6-release-notes.adoc:153]: @ousleyp Yes, it is still relevant. Soon this functionality will be removed from CNV and that is why the BZ was closed as `WONTFIX`.

File: virt/virt-2-6-release-notes.adoc (hunk 1 above)
Comment (stu-gott) [reply virt/virt-2-6-release-notes.adoc:95]: yes. please keep this release note for 2.6. The BZ for this is still open.

File: virt/virt-2-6-release-notes.adoc (hunk 3 above)
Comment (stu-gott) [reply virt/virt-2-6-release-notes.adoc:149]: This release note can be dropped for 2.6. We now honor node drains.

File: virt/virt-2-6-release-notes.adoc (hunk 2 above)
Comment (tiraboschi) [reply virt/virt-2-6-release-notes.adoc:104]: https://bugzilla.redhat.com/1796410 is an OCP UI bug and it's still on assigned, so yes, we should keep the note.

Comment (ousleyp) [reply virt/virt-2-6-release-notes.adoc]: @tiraboschi, please check the changes to the patch command below the note when you get the chance. Thanks!

**Review by aburdenthehand — APPROVED** [review]
lgtm

**Pull request merged**